webserver/templates/datasets/list.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html' %}
+
+{% block title %}Public datasets{% endblock %}
+
+{% block content %}
+    <h3>Public datasets</h3>
```

Comment:
would be good to have an option to filter by *running*, *in queue*, *finished*, *new*

If no filter, put in running and in queue first

We will need paging before we get too many datasets